nexus-repository-conan/src/main/java/org/sonatype/repository/conan/internal/hosted/v1/HostedHandlers.java

```diff
@@ -48,14 +55,30 @@
   public static final Handler uploadUrl = context -> {
     State state = context.getAttributes().require(State.class);
     ConanCoords coord = ConanHostedHelper.convertFromState(state);
-    String downloadUrlAsJson = context.getRepository()
-        .facet(ConanHostedFacet.class)
-        .getDownloadUrlAsJson(coord);
+    ObjectMapper objectMapper = new ObjectMapper();
```

Comment:
Minor: `ObjectMapper` can be a heavier class. Consider adding a private static field `private static ObjectReader objectReader = new ObjectMapper().reader();` and use the reader directly, which is thread safe a lighter weight. Might not be a big deal if this handler is not expected to be used frequently.

Also using the reader I don't think you'll need the intermediate `TypeReference` to read the value.

Reply:
Thanks, fixed 25edb0efc472f8b9787965230222b039736cda15